### DISCUSSION TITLE
HTTP Status 403 – Forbidden

### DISCUSSION THREAD

**Discussion by MahajanTarun** [OxalisCommunity/oxalis#524] [Q&A]
Using Tomcat, I am trying to run Oxalis.
But after following all the installation steps, I am unable to run Oxalis in Tomcat.

The Manager-GUI shows that Oxalis is running, but when I go into the path(as shown in the images), it returns 403 http error.

I tried a lot figuring it out if I am missing any installation steps.

Please guide me on the issue.
![oxalis_error1](https://user-images.githubusercontent.com/16743172/75887045-343c0f00-5e4f-11ea-9847-9d8f2d2947a8.png)
![oxalis_error2](https://user-images.githubusercontent.com/16743172/75887051-3605d280-5e4f-11ea-870d-25ebc190a0dc.png)

**Comment by SuperJuell** [comment]
We will update the documentation for this.
